Library/ViewModels/PledgeViewModel.swift

```diff
@@ -405,22 +403,25 @@ public class PledgeViewModel: PledgeViewModelType, PledgeViewModelInputs, Pledge
 
     // MARK: - Apple Pay
 
-    let applePayButtonTappedAndIsChangingPaymentMethod = self.applePayButtonTappedSignal
-      .filter { _ in !isNativeRiskMessagingControlEnabled() }
+    let changingApplePayPaymentMethod = self.applePayButtonTappedSignal
       .combineLatest(with: context)
       .map(second)
       .filter { $0 == .changePaymentMethod }
       .ignoreValues()
 
-    // If the Optimizely risk messaging experiment is set to the control AND the Pay With Apple button is tapped
-    // Or if the Optimizely risk messaging experiment is set to the variant, and we are changing the payment method to Pay With Apple
-    // Or if the Optimizely risk messaging experiment is set to the variant and it is dismissed, this emits
-    let applePayButtonTappedOrRiskMessagingModalDismissed = Signal.merge(
-      self.applePayButtonTappedSignal.filter(isNativeRiskMessagingControlEnabled),
-      applePayButtonTappedAndIsChangingPaymentMethod,
-      self.riskMessagingViewControllerDismissedProperty.signal.skipNil().filter(isTrue).ignoreValues()
+    let applePayButtonTappedAndOrIsChangingPaymentMethod = Signal.merge(
```

Comment:
I think this entire signal should just be `self.applePayButtonTappedSignal` instead (since we're always in the control of the experiment now). The way it's currently written, if the apple pay button is tapped in the change pledge context, the signal looks like it'll emit twice.

Reply:
I hear you. There are XCTAssertEquals assertions in the tests for this that compare arrays, so if it was emitting twice, I would think those would catch it. 

I think you're suggestion reads more clearly anyway tho. I'll add this to the next commit.